turbopy/diagnostics.py

```diff
@@ -131,6 +132,13 @@ def initialize(self):
         as an instance of the :class:`CSVOuputUtility` class.
         """
         # set up function to interpolate the field value
+        if "directory" in self._input_data:
+            d = Path(self._input_data["directory"])
+        else:
+            # Set a default output directory
+            d = Path("default_output")
+            self._input_data["directory"] = str(d)
+        d.mkdir(parents=True, exist_ok=True)
```

Comment:
This code should not be here explicitly. You should just call `initialize` from the base class.
```suggestion
        super().initialize()
```

Reply:
Note that I didn't test that code, but I think that should be the correct way to call `initialize` from the base class. See https://realpython.com/python-super